zipline-loader/src/test/kotlin/app/cash/zipline/loader/TopologicalSortTest.kt

```diff
@@ -41,8 +57,20 @@ class TopologicalSortTest {
     }
   }
 
+  private fun assertTopologicalSort(unsorted: List<String>, sorted: List<String>, vararg edges: String) {
+    val sourceToTarget = edges(*edges)
+    assertThat(unsorted.isTopologicallySorted(sourceToTarget)).isFalse()
+    assertThat(sorted.isTopologicallySorted(sourceToTarget)).isTrue()
+
+    val actual = unsorted
+      .topologicalSort(sourceToTarget)
+
+    assertThat(actual.isTopologicallySorted(sourceToTarget)).isTrue()
```

Comment:
This is redundant with line 63?

Reply:
L63 asserts that the sorted "expected" list is actually sorted. L68 checks that the sorted output is sorted. Probably redundant but trying to catch any possible unsorted issue